addons/sourcemod/scripting/tfgo/config.sp

```diff
@@ -36,6 +44,33 @@ public void ReadWeaponConfig(KeyValues kv)
 	}
 }
 
+public void ReadGearConfig(KeyValues kv)
+{
+	if (kv.JumpToKey("Gear", false))
+	{
+		if (kv.GotoFirstSubKey(false))
+		{
+			do
+			{
+				char id[PLATFORM_MAX_PATH];
+				kv.GetSectionName(id, sizeof(id));
+				
+				Gear gear;
+				gear.id = StringToInt(id);
+				gear.cost = kv.GetNum("cost", -1);
+				
+				int length = g_AvailableGear.Length;
+				g_AvailableGear.Resize(length + 1);
+				g_AvailableGear.Set(length, gear.id, 0);
+				g_AvailableGear.SetArray(length, gear, sizeof(gear));
```

Comment:
`g_AvailableGear.Set` seems pointless to do when `g_AvailableGear.SetArray` does same thing but setting everything. And even then, all of those could've been replaced to `g_AvailableGear.PushArray`